tests/Directory.Build.props

```diff
@@ -3,7 +3,6 @@
 	<PropertyGroup>
 		<TargetFrameworks>net8.0;net462</TargetFrameworks>
 		<IsTestProject>true</IsTestProject>
-		<Nullable>disable</Nullable>
```

Comment:
Could we also add `<TreatWarningsAsErrors>true</TreatWarningsAsErrors>` so that any future warnings now break the build?

Reply:
[Added](https://github.com/microsoft/kiota-dotnet/pull/329/commits/2b5ac5fa883759f24f4a9b1cec3b0fd8f1ded82b)